pages/signup/app.js

```diff
@@ -39,40 +41,58 @@ import { validationUsers } from "../../utils/validationUsers.js";
       });
     });
 
+    emailInput.addEventListener("focusin", () => {
+      alreadyEmailValidation = false;
+    });
+
     emailInput.addEventListener("focusout", () => {
-      // const userInput = emailInput.value.trim();
-      currentUser.email = validationUserEmail(
+      if (alreadyEmailValidation) return;
+
+      CURRENT_INPUT.email = validationUserEmail(
         emailInput.value.trim(),
         "signup"
       );
       emailInput.style.color = "#3e3e43";
```

Comment:
(이번 작업과는 관계 없지만)
```suggestion
      emailInput.classList.add = "validated";
```
이런식으로 class를 활용하는게 좀 더 명식적일 것 같아요.